eval/eval.go

```diff
@@ -34,19 +34,21 @@ const (
 )
 
 const (
-	defaultValuePrefix = "▶ "
-	initIndent         = vals.NoPretty
+	defaultValuePrefix        = "▶ "
+	defaultNotifyBgJobSuccess = false
```

Comment:
Default to `true` :)

Reply:
I should really not try and code with a toddler and infant awake...